### PR TITLE
Fix issue for create test report when errata is not ready

### PR DESCRIPTION
**Pull request by wewang58**
@rioliu-rh  please check it, thanks
when create 4.13.7 report, which errata is not ready, 
original output is:
$oar -r 4.13.7 create-test-report
'4.13.7'

so fixed it, when errata is not ready, should give info about the reason, 
fixed output:
$oar -r 4.13.7 create-test-report
2023-07-27T13:11:58Z: ERROR: config store initialization failed
Traceback (most recent call last):
  File "/home/wewang/522/release-tests/oar/cli/cmd_group.py", line 46, in cli
    ConfigStore(release)
  File "/home/wewang/522/release-tests/oar/core/config_store.py", line 58, in __init__
    raise ConfigStoreException(f"[{self.release}] advisories are not ready, cannot create test report,you can check:{url}")
oar.core.exceptions.ConfigStoreException: [4.13.7] advisories are not ready, cannot create test report,you can check:https://raw.githubusercontent.com/openshift-eng/ocp-build-data/openshift-4.13/releases.yml

### CODE REVIEW

File: oar/core/worksheet_mgr.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def create_test_report(self):
             # update advisory info
             ad_cell_value = ""
             for k, v in self._cs.get_advisories().items():
-                ad_cell_value += f"{k}: {v}\n"
+                ad_cell_value += f"{k}: https://errata.devel.redhat.com/advisory/{v}\n"
```
Comment (rioliu-rh) [inline oar/core/worksheet_mgr.py:87]: reject this change

File: oar/cli/cmd_group.py [hunk 2]
```diff
@@ -43,10 +43,10 @@ def print_version(ctx, param, value):
 def cli(ctx, release, debug):
     util.init_logging(logging.DEBUG if debug else logging.INFO)
     try:
-        cs = ConfigStore(release)
+        ConfigStore(release)
```
Comment (rioliu-rh) [inline oar/cli/cmd_group.py:46]: try/except block can be removed.

File: oar/core/config_store.py [hunk 3]
```diff
@@ -52,8 +52,10 @@ def __init__(self, release):
                 self._build_data = yaml.safe_load(response.text)
             except yaml.YAMLError as ye:
                 raise ConfigStoreException("ocp build data format is invalid") from ye
-
-        self._assembly = self._build_data["releases"][self.release]["assembly"]
+        if self.release in self._build_data["releases"]:
+            self._assembly = self._build_data["releases"][self.release]["assembly"]
+        else:
+            raise ConfigStoreException(f"[{self.release}] advisories are not ready, cannot create test report, you can check:{url}")
```
Comment (rioliu-rh) [inline oar/core/config_store.py:58]: description is not accurate, if `release` cannot be found in `releases`, it means not only advisory is not ready, all the build data is missed.

Comment (wewang58) [reply oar/core/config_store.py:58]: {self.release}] advisories  and build data are not ready?

Comment (rioliu-rh) [reply oar/core/config_store.py:58]: all the data are not ready. not only ad

File: oar/core/config_store.py [hunk 4]
```diff
@@ -52,8 +52,10 @@ def __init__(self, release):
                 self._build_data = yaml.safe_load(response.text)
             except yaml.YAMLError as ye:
                 raise ConfigStoreException("ocp build data format is invalid") from ye
-
-        self._assembly = self._build_data["releases"][self.release]["assembly"]
+        if self.release in self._build_data["releases"]:
+            self._assembly = self._build_data["releases"][self.release]["assembly"]
+        else:
+            raise ConfigStoreException(f"[{self.release}] advisories and build data are not ready, cannot create test report, you can check:{url}")
```
Comment (rioliu-rh) [inline oar/core/config_store.py:58]:
```suggestion
            raise ConfigStoreException(f"[{self.release}] build data is not ready, cannot create test report, you can check:{url}")
```

Comment (wewang58) [reply oar/core/config_store.py]: Updated

**Pull request merged**